builder/src/bin/permissionless-builder.rs

```diff
@@ -123,7 +122,7 @@ async fn main() -> anyhow::Result<()> {
         genesis.chain_config,
         l1_params,
         opt.state_peers,
-        <SeqTypes as NodeType>::Base::instance(),
+        StaticVersion::<0, 1>::instance(),
```

Comment:
Why change from using `SeqTypes` to hard-coding?

Reply:
I think  builder uses v1 api so I added V1 version. I can change it to Base and see if tests passes

Reply:
Actually, now I remember I was getting version mismatch errors and had to change to v1

Reply:
Oh builder API version is unrelated to sequencer protocol...fair enough